common/source/docs/common-hitec-uavcan-servos.rst

```diff
@@ -0,0 +1,204 @@
+
+-   First step is to update the following two fields to -45.00 and 45.00 
+
+   .. image:: ../../../images/hitec-uavcan-servos-config3.png
+       :width: 450px
+
+-   Press that section’s [Left] button to swing the servo to its “Min” value and note the number it displays, in this case 6144
+-   Next press the [Right] button and note the value it displays, in this case 10240 
+-   In the “Servo Configuration” section, select the “Mode” tab and enter those values for the “POSITION MAX_LIMIT” and “POSITION MIN_LIMIT” fields, press each of their corresponding [SET] buttons, press [SAVE] and finally reboot the servo by pressing the [SERVO RESET] (Reminder, it probably just says [SERVO] on your screen like above)
+
+   .. image:: ../../../images/hitec-uavcan-servos-config4.png
+       :width: 450px
+
+-   When the servo reboots, use the left/right buttons under the word "Test" to command unitless values of -1.000 and +1.000 to validate that those values are giving -45 and +45 degrees of servo travel.  Using these left/right buttons simulate ArduPilot sending full-swing commands in unitless values of -1 to +1.  Note: You may be required to update the "CAN ID" and "SERVO ID" under the DPC-CAN Configuration section of the app so that the app is now talking to the servo on its new Servo ID, as well as from CAN ID Node 10.
+
+   .. image:: ../../../images/hitec-uavcan-servos-config6.png
+       :width: 450px
+
+You can now move to ArduPilot Configuration.
+
+ArduPilot Configuration for UAVCAN Servos
+=========================================
+This procedure will cover AutoPilot configuration of the Hitec UAVCAN Servo.  This document is a continuation of the previous document and comes with the following assumptions:
+-   ArduPilot will be NODE 10 on the UAVCAN Bus
+-   Servo will be Servo 2, which is going to be configured as the Elevator servo
+-   The AutoPilot software in this document is ArduPlane 4.0.5
+-   The AutoPilot hardware in this instance is a CUAV v5 Nano and we will be connected to CAN1; any ArduPilot hardware with CAN broken out will be configured in functionally the same way
+
+All of the hardware AutoPilot devices currently supported in ArduPilot do not provide suitable power to the CAN ports.  Because of this, it is required that you supply appropriate current to the UAVCAN servos outside of that bus.  Hitec has a collection of breakout boards to meet this demand and simplify wiring.  Do not forget to connect a bus termination resistor if you are not connecting any other devices.
+
+After connecting to ArduPilot, the following parameter changes are required:
+
+``CAN_D1_PROTOCOL`` =1
```

Comment:
all these should be param hyperlinks instead of shadowboxed...and single spaced bullet lists